src/libs/Notification/PushNotification/backgroundRefresh/index.android.js

```diff
@@ -0,0 +1,40 @@
+import Onyx from 'react-native-onyx';
+import * as App from '../../../actions/App';
+import Visibility from '../../../Visibility';
+import ONYXKEYS from '../../../../ONYXKEYS';
+
+function getLastOnyxUpdateID() {
```

Comment:
Can this ever fail? Just wondering if we need to check that `lastUpdateID` is not empty when we pass it to reconnectApp

Reply:
The new reconnectApp already handles falsy values [here](https://github.com/Expensify/App/pull/23516/files#diff-7ba339cca3eda747a4a803353b2b9ec61d991ae1d3d52ee4726aa0d659308e1aR201-R211) so we should be good if `lastUpdateID` is not set